tensorflow/contrib/learn/python/learn/estimators/linear.py

```diff
@@ -243,7 +243,7 @@ def sdca_model_fn(features, labels, mode, params):
 
   parent_scope = "linear"
 
-  with variable_scope.variable_op_scope(
+  with variable_scope.variable_scope(
```

Comment:
```
with variable_scope.variable_scope(
        values=features.values(), name_or_scope=parent_scope) as scope:
```

Reply:
Is this what you mean? Finding it quite hard to get to grips with this huge repo haha. Can't find any functions that take a parameter `name_or_scope`?

Reply:
I tell a lie. I have just seen - https://github.com/tensorflow/tensorflow/blob/0d3713e808ac27b547619bd850713eb28ff26eb4/tensorflow/tools/api/golden/tensorflow.variable_scope.pbtxt#L7

Reply:
Yes, the snippet in your first comment looks right to me. The arguments are just permuted slightly. variable_scope arguments are here: https://github.com/tensorflow/tensorflow/blob/1897cea1807c28d998b7efc3906de48d75d99a6d/tensorflow/python/ops/variable_scope.py#L1858

Reply:
Weird. PyCharm did not pick that up.